pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -587,6 +595,15 @@ func (ctrl *Controller) syncContainerRuntimeConfig(key string) error {
 		klog.V(4).Infof("Finished syncing ContainerRuntimeconfig %q (%v)", key, time.Since(startTime))
 	}()
 
+	// create the MC for the drop in default-container-runtime crio.conf file
+	if err := ctrl.createDefaultContainerRuntimeMC(); err != nil {
```

Comment:
So, this creates `99-$POOL-generated-crio-default-container-runtime` objects, which technically has a potential race condition here. Since:
1. it's not actually generated by the controller (no generated-by-controller-version stamp), and
2. even if it did, the MCO has a blind spot today for new generated MCs by subcontrollers upon upgrades

Such that if the render controller sees that all other MCs had updated, it will generate a new render config before this actually gets to get created, before generating a second render shortly after when this gets created. This isn't too bad since the likely outcome there is an extra reboot for a singular node per pool (not super noticeable but some users have called it out). And really that shouldn't happen unless the first attempt fails due to some blip.

All that's to say we should be aware that this is a rare but possible "unexpected behaviour" but given our options I don't think there's really a much better way to do this.